package-lock.json

```diff
@@ -2353,6 +2353,157 @@
                 "regenerator-runtime": "^0.13.3"
             }
         },
+        "@livechat/customer-auth": {
```

Comment:
Why do we need these changes in `package-lock.json`?

Reply:
These were the missing dependencies from master branch. I've now removed it and pulled in the latest changes after the latest merged by Amam.